cli/src/cli_util.rs

```diff
@@ -1272,7 +1299,16 @@ to the current parents may contain changes from multiple commits.
         locked_ws.finish(repo.op_id().clone())?;
         self.user_repo = ReadonlyUserRepo::new(repo);
 
-        self.maybe_snapshot(ui)
+        let stats = self
+            .maybe_snapshot_impl(ui)
+            .map_err(|err| err.into_command_error())?;
+        print_snapshot_stats(
+            ui,
+            &stats,
+            self.env().path_converter(),
+            SnapshotContext::Automatic,
+        )?;
+        Ok(stats)
```

Comment:
Perhaps, this `stats` will be printed twice?

Reply:
Ah yes you're right! Nice catch

Reply:
Fixed!